pkg/gitmanager/setup.go

```diff
@@ -171,10 +171,13 @@ func (gm *gitManager) initGit(ctx context.Context, payload *core.Payload, oauth
 	return nil
 }
 
-func (gm *gitManager) getUnzippedFileName(gitProvider, orgName, repoName, forkSlug, commitID string) string {
+func (gm *gitManager) getUnzippedFileName(gitProvider, orgName, repoName, forkSlug, commitID string, privateRepo bool) string {
 	switch gitProvider {
 	case core.GitHub:
-		return orgName + "-" + repoName + "-" + commitID
+		if privateRepo {
+			return orgName + "-" + repoName + "-" + commitID
+		}
+		return orgName + "-" + repoName + "-" + commitID[:7]
```

Comment:
what is this magic number 7?

Reply:
Commit small sha length in github

Reply:
Why we are using the whole commit in private repo then 
Is there any restriction added by github

Reply:
Github zip ball API is giving full commit in private repos unzipped file name and shorthand commit in public, That's what I observed